amr-wind/wind_energy/actuator/wing/fixed_wing_ops.H

```diff
@@ -68,16 +88,64 @@ struct ReadInputsOp<FixedWing, ActSrcLine>
             amrex::max(max_eps, max_epsc) * max_chord * 3.0;
         const auto& p1 = wdata.start;
         const auto& p2 = wdata.end;
-        // clang-format off
-        info.bound_box = amrex::RealBox(
-            amrex::min(p1.x(), p2.x()) - search_radius,
-            amrex::min(p1.y(), p2.y()) - search_radius,
-            amrex::min(p1.z(), p2.z()) - search_radius,
-            amrex::max(p1.x(), p2.x()) + search_radius,
-            amrex::max(p1.y(), p2.y()) + search_radius,
-            amrex::max(p1.z(), p2.z()) + search_radius
-        );
-        // clang-format on
+        // Set up bounding box depending on motion type
```

Comment:
I guess the reason for the integers is the switch statement here.
Was there a reason to use the switch statements instead of if statements?
If you leave the switch statements (they are fine) can you please add a comment to each case to explain what they are, so that it makes it easier to read the code (non, linear, sine, etc).